lesson_07/conditionals/src/lesson7.ts

```diff
@@ -67,7 +115,14 @@ export function addNumbers(values: number[]): number {
  * @return An array containing the first `n` Fibonacci values.
  */
 export function getFirstNFibonacciNumbers(n: number): number[] {
-  return [];
+  
+  const array = [n]
```

Comment:
You probably don't want to put `n` here.  If the function is called with `99`, then the array will be initialized to `[99]` which is not what you want.

Reply:
changed to start the array as [1, 1] instead of [n]